Sources/SwiftBuildSupport/PackagePIFBuilder+Helpers.swift

```diff
@@ -793,110 +798,128 @@ extension TSCUtility.Version {
 
 // MARK: - Swift Build PIF Helpers
 
-/// Helpers for building custom PIF targets by `PIFPackageBuilder` clients.
-extension SwiftBuild.PIF.Project {
+extension ProjectModel.BuildSettings {
+    subscript(_ setting: MultipleValueSetting, default defaultValue: [String]) -> [String] {
```

Comment:
Now I'm no longer sure if this syntax sugar is such a great idea.... note that it doesn't make it explicit if the `default` value is prepended or appended (it's the former).
```swift
var settings = ProjectModel.BuildSettings() 
settings[.PATH, default: "/bin"] + ["/foo/bar"]
// PATH is now "/bin:/foo/bar"
```

Reply:
This seems OK to me, I think the syntax here reads as if it implies appending